### DISCUSSION TITLE
How to detect 'mousewheel' event?

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [pixijs/pixijs#7692] [Q&A]
I wish there is... 

container.on('mousewheel', () => )

How can I use mousewheel in pixi app?

**Comment by bigtimebuddy** [comment]
There are a few [past issues](https://github.com/pixijs/pixijs/issues?q=is%3Aissue+mousewheel+) about this. Read through those and you'll probably find some useful workarounds.

**Comment by miltoncandelero** [comment]
You can always jump pixi entirely and hook yourself to the dom.
`document.addEventListener('wheel', yourWheelFunction);`

The downside is that no matter where the pointer is on your screen when you wheel your function will be called.
